spec/slack_transformer/html/paragraph_spec.rb

```diff
@@ -21,53 +21,44 @@
         end
       end
 
-      context 'Hello<p>World</p>Again' do
-        let(:input) { 'Hello<p>World</p>Again' }
-
-        it 'return World and Again on a newline' do
-          expect(transformation.to_slack).to eq("Hello\nWorld\nAgain")
-        end
-      end
-     
-      context 'Hello<p>World</p>' do
-        let(:input) { 'Hello<p>World</p>' }
-        
-        it 'return World on a newline' do
-          expect(transformation.to_slack).to eq("Hello\nWorld")
-        end
-      end
```

Comment:
Just please confirm with Alex that we don't want this, as it's a valid HTML test
<img width="324" alt="image" src="https://github.com/affinity/slack_transformer/assets/114527410/6fe661df-110a-4807-bb24-8faa1d6073e9">